core/client/fs/src/main/java/alluxio/client/block/stream/LocalFilePacketWriter.java

```diff
@@ -95,29 +131,52 @@ public void close() {
     if (mClosed) {
       return;
     }
-    try {
-      mWriter.close();
-    } finally {
-      mClosed = true;
-    }
+    mClosed = true;
+
+    mCloser.register(new Closeable() {
+      @Override
+      public void close() throws IOException {
+        NettyRPC.call(mNettyRPCContext, new ProtoMessage(
+            Protocol.LocalBlockCompleteRequest.newBuilder().setBlockId(mBlockId)
+                .setSessionId(mSessionId).build()));
+      }
+    });
```

Comment:
Is there a difference between registering this with the closer and doing it explicitly (can save some boilerplate) since we are closing the closer immediately afterward?